k-distribution/tests/regression-new/Makefile

```diff
@@ -32,7 +32,7 @@ SUBDIRS=issue-2273 \
 	kore-brackets \
 	quadratic-poly-unparsing \
 	io-java \
-	symbolic-int-function \
+	symbolic-int-function
```

Comment:
this was done deliberately in order to simplify diffs that modify this file, which frequently conflict with one another. I'd appreciate it if it was kept the way it is now.

Reply:
Makes sense, fixed.